### ISSUE TITLE
Query_region example in Simbad docs doesn't work

### ISSUE THREAD

**Issue by dkagrawal** [astropy/astroquery#848]
http://astroquery.readthedocs.io/en/latest/simbad/simbad.html#query-a-region
```
from astroquery.simbad import Simbad
result_table = Simbad.query_region("sirius")
print(result_table)
```
Here, the problem arises because query_region only accepts co-ordinates as its first argument and fails to parse it in case of names. The error looks like this - 
```
ValueError: (u'Cannot parse first argument data "sirius" for attribute ra', ValueError(u"Syntax error parsing angle 'sirius'",))
```

**Comment by keflavich** [comment]
Yes, this is both a documentation error and probably a regression; I believe we previously attempted to lookup coordinates for object names.

**Comment by bsipocz** [comment]
It seems that there is still a bit of cruft around, and maybe we can remove even more of the coordinate parsing stuff and use astropy core by default?

For the sake of the documentation, maybe it's best to remove the name lookups and use the coordinates?

**Comment by keflavich** [comment]
De-crufting will take work.  I'm in favor of fixing the documentation first, then more deeply investigating what to do about coordinate parsing.  I like the idea of allowing object names to be passed to `SkyCoord.from_name`, but perhaps we should not allow that and instead document it as:

```
Simbad.query_region(SkyCoord.from_name('sirius'))
```
etc.

**Comment by bsipocz** [comment]
For the sake of a working docs, I would even go for 
``Simbad.query_region("06 45 08.917 -16 42 58.02")`` and mention that this is the coord of Sirius.

**Comment by bsipocz** [comment]
@dkagrawal - The bottom line, is that it would be great if you could fix the docs. 
And later, if you like, there is more work to be done with this issue.

**Comment by dkagrawal** [comment]
@bsipocz Can't we get co-ordinates from the table that we got from `Simbad.query_object(<name>)` instead of `SkyCoord.from_name` ?

**Comment by dkagrawal** [comment]
@bsipocz @keflavich Maybe Something like this -
```
try:
        ra, dec, frame = _parse_coordinates(coordinates)
except ValueError:
        base_query_str = base_query_str.replace("coo","around")
        ra, dec, frame = ['',coordinates,'icrs']
```
so that the script sent from `_args_to_payload` is changed?

**Comment by keflavich** [comment]
@dkagrawal When you first posted this, I did notice an error, but I dug through the code again and see that actually, everything looks fine.  Re-running the "buggy" query above, it now works locally.  I *think* this indicates that the problem was a failed [`NameResolver`](https://github.com/astropy/astroquery/blob/master/astroquery/utils/commons.py#L183) query, which probably indicates that we both hit a time when the name resolver server was down.

So, in other words, we can probably close this issue, assuming the original queries now work again.
